core/src/main/kotlin/no/strooped/StroopedController.kt

```diff
@@ -5,30 +5,39 @@ import com.badlogic.gdx.Screen
 import com.badlogic.gdx.utils.ObjectMap
 import no.strooped.service.GameLifecycleService
 import no.strooped.service.JoinGameService
+import no.strooped.service.SocketService
+import no.strooped.view.screen.EndRoundScreen
 import no.strooped.view.screen.JoinGameScreen
 import no.strooped.view.screen.LobbyScreen
+import no.strooped.view.screen.TaskScreen
+import no.strooped.view.screen.components.ColorButton
 
 const val TITLE = "Strooped"
 
 class StroopedController : Game() {
     private val screens: ObjectMap<Class<out Screen>, Screen> = ObjectMap()
-    var joinGameService: JoinGameService = JoinGameService(socketService = Any())
-    var gameLifecycleService: GameLifecycleService = GameLifecycleService()
+    var socket: SocketService = SocketService()
+    var joinGameService: JoinGameService = JoinGameService(socket)
+    var gameLifecycleService: GameLifecycleService = GameLifecycleService(socket)
 
     override fun create() {
         loadScreens()
         inititalizeServices()
-        changeScreen(JoinGameScreen::class.java)
+        screens.put(EndRoundScreen::class.java, EndRoundScreen(this))
+        changeScreen(EndRoundScreen::class.java)
```

Comment:
Won't this start the application on the EndRoundScreen?

Reply:
Yes, I left it there so you can take a look at the screen if you wanted to